modules/oadp-release-notes-1-3-1.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes-1-3.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-release-notes-1-3-1_{context}"]
+= OADP 1.3.1 release notes
+
+The {oadp-first} 1.3.1 release notes lists new features and resolved issues.
+
+[id="new-features-1-3-1_{context}"]
+== New features
+
+.OADP 1.3 Data Mover is now fully supported
+
+The OADP built-in Data Mover, introduced in OADP 1.3 as a Technology Preview, is now fully supported for both containerized and virtual machine workloads.
+
+.{ibm-cloud-name} S3 backup storage provider is now supported
```

Comment:
```suggestion
.{ibm-cloud-name} is now supported as an AWS S3 compatible backup storage provider.
```

Reply:
S3 is generally attached to AWS. I believe it should just be "{ibm-cloud-name}"
Please confirm if the term "{ibm-cloud-name} S3 " is valid.

Reply:
IBM do offer `IBM Cloud Object Storage S3`, see [HERE](https://www.ibm.com/docs/en/watson-orchestrate?topic=storage-cloud-object-s3)

Reply:
@anarnold97 @xenolinux , could you please confirm if the following doc-text is ok for this-

> .{ibm-cloud-name} S3 is now supported as an AWS compatible backup storage provider
> 
> Previously, the AWS compatible object storage provider, {ibm-cloud-name} S3, was unsupported as backup storage locations.
> With this update, {ibm-cloud-name} S3 is now supported as an AWS compatible backup storage provider.

Thanks in advance!

Reply:
@PrasadJoshi12 && @amastbau  - please can we have a QE opinion on this
Thanks

Reply:
@apurvabhide17 looks good. Sorry I am not sure about this terminology.  I would just confirm with QE or SME if it should be "IBM Cloud Object Storage S3"?
In preview it's appearing as "IBM Cloud® S3"

Reply:
+1 for IBM cloud Object Storage

Reply:
Hello @anarnold97, @xenolinux, @PrasadJoshi12 ,  
To  add "IBM cloud Object Storage" and avoid the confusion related to "the AWS compatible object storage provider, {ibm-cloud-name} S3, was unsupported", sharing my final draft here-

```
.{ibm-cloud-name} Object Storage is now supported as a backup storage provider

{ibm-cloud-name} Object Storage is one of the AWS S3 compatible backup storage provider, which was unsupported previously.  
With this update, {ibm-cloud-name} Object Storage is now supported as an AWS S3 compatible backup storage provider.
```
Please let me know so that I can add it and the PR will be ready for the merge process. Thanks!

Reply:
LGTM - docs wise

Reply:
@apurvabhide17 I think one change is reverted from the very initial suggestion. 

The phrase "one of the" is always followed by a plural noun. So, it should be "...one of the AWS S3 compatible backup storage **providers**"

Reply:
yup, my bad, I've updated it. Thanks :)